### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by Vicx95**

### CODE REVIEW

**Comment by ziobron** [comment]
I will put some comments here to check automation. Sorry for emails you will get ;)

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,28 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    int result = 0;
```
Comment (ziobron) [inline module1/homework/calculate/calculate.hpp:6]: Just a random comment, please ignore it ;)

**Comment by ziobron** [comment]
+5 XP

**Comment by ziobron** [comment]
+5 XP

**Comment by ziobron** [comment]
+5 XP

**Comment by ziobron** [comment]
Some random comment

**Comment by ziobron** [comment]
+5 XP

**Comment by ziobron** [comment]
+1 XP

**Comment by ziobron** [comment]
+4 XP

**Comment by ziobron** [comment]
+2 XP za punktualność

**Comment by ziobron** [comment]
+2 XP za punktualność

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,28 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    int result = 0; 
+    if(command == "add")
+    {
+        result = first + second;
```
Comment (Kasperice) [inline module1/homework/calculate/calculate.hpp:9]: A bit shorter way is to return value in a way ```return std::to_string(first + second);```

**Comment by ziobron** [comment]
rid:1429 - calculate +5XP

**Comment by ziobron** [comment]
rid:1119 - punktualność +2XP

**Pull request closed without merging**